R/network_analysis.R

```diff
@@ -41,7 +41,8 @@ make_res <- function(dat, period_select = NULL,
   if(period_select %in% dat$period == F){
     stop("period_select not in period of data")
   }
-  if(is.null(sector_select) == F){
+ browser()
```

Comment:
I believe browser() pauses execution and is a debugging function? I haven't used it myself, but I suspect it shouldn't be in the published code.

Reply:
Thanks @todd-davidson-beis Well spotted. I was working too quickly yesterday. That is now fixed. Please do let me know any other issues you spot